multi-tenancy-observability/README.md

```diff
@@ -19,28 +19,49 @@ summary: How Giant Swarm manages multi-tenancy to isolate observability data per
 
 ## Introduction
 
-Giant Swarm managed components produce a lot of observability data (metrics, logs, traces and so on).
-For instance, we have reached our ingestion limits in Prometheus more than a year ago.
-On top of this, as part of the developer platform, we want to be able to ingest all sorts of customer observability data and provide data isolation between tenants.
+Moving towards our observability platform, we have to consider to important topics.
+
+First, we generate a lot of observability data.
+For instance, we reached our ingestion limits in Prometheus more than a year ago and each new component we add brings a lot more metrics so we have to be mindful about the metrics we keep.
+This issue got a whole lot bigger when we added logging with Loki to the mix.
+
+Second, our product presentlyallows anyone with access to Grafana on the management clusters to access all the data (metrics and logs) for all workload clusters without any kind of data seggregration. This is part of the reason why only a subset of our customers (generaly only the platform teams) have access to our managed grafana and is also why our shared installation grafana is not accessible to customers.
+
+
+As part of the developer platform, we want to be able to ingest all sorts of customer observability data. This will require Giant Swarm observability platform to be able to provide data isolation between tenants.
```

Comment:
```suggestion
As part of the developer platform, we want to be able to ingest all sorts of customer observability data. For example a lot of customers are interested in monitoring of application developers workloads. This will require Giant Swarm observability platform to be able to provide data isolation between tenants, so that customer teams have limited access to the data originating on clusters they own.
```